crates/gateway-config/src/telemetry/mod.rs

```diff
@@ -19,7 +19,7 @@ pub use exporters::{BatchExportConfig, OpenTelemetryExportersConfig, StdoutExpor
 #[serde(default, deny_unknown_fields)]
 pub struct TelemetryConfig {
     /// The name of the service
-    pub service_name: String,
+    pub service_name: Option<String>,
```

Comment:
Would add the default here. I find it way too deep otherwise, risking inconsistencies between traces/metrics/etc. Just need to implement `Default` for TelemetryConfig as we have `#[serde(default)]`

Reply:
I don't want to put that much effort in.

Reply:
I mean the times, it takes to have this whole discussion and writing the unwrap, you would have written the `Default` :D  but anyway, approved

Reply:
fair enough :D

Reply:
Thank you!